IBAnimatableApp/BlurEffectViewController.swift

```diff
@@ -28,7 +28,7 @@ class BlurEffectViewController: UIViewController {
       .then(.slide(way: .in, direction: .left))
       .then(.pop(repeatCount: 1))
       .completion {
-        self.blurEffectView.alpha = 0
+        print("Done ;)")
```

Comment:
Is that just for debug, if not I think we can find a better feedback to show the completion